disable finding Boost libraries in the system to force use of bundled Boost
author: justbennet
--- bcl2fastq/src/cmake/boost.cmake.orig	2021-05-29 13:08:27.575571488 -0400
+++ bcl2fastq/src/cmake/boost.cmake	2021-05-29 12:17:10.140267869 -0400
@@ -101,7 +101,7 @@
     set(Boost_USE_STATIC_LIBS ON) 
 endif (BCL2FASTQ_FORCE_STATIC_LINK)
 
-find_package(Boost ${BCL2FASTQ_BOOST_VERSION} COMPONENTS ${BCL2FASTQ_BOOST_COMPONENTS})
+# find_package(Boost ${BCL2FASTQ_BOOST_VERSION} COMPONENTS ${BCL2FASTQ_BOOST_COMPONENTS})
 #
 # If the right version of boost is not found, it will be built from the distribution
 #
